src/AppBundle/Controller/UserController.php

```diff
@@ -41,4 +41,40 @@ public function getAction()
     {
         return $this->handleGet();
     }
+
+
+    /**
+     * @Route("/user/{id}", name="user_delete")
+     * @Method("DELETE")
+     *
+     * @return [type] [description]
+     */
+    public function deleteAction()
+    {
+        // get user id from the request
+
+        // get the entity manager out of the container
+        $em = $this->getDoctrine()->getManager();
```

Comment:
this can be just $this->getEntityManager() since its available in CarbonApiController